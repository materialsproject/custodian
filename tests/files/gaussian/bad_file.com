%chk=Checkpoint.chk
%mem=48GB
%NProcShared=28
#P PBE1PBE/6-31+G* opt scf=(maxcycle=100) guess=read geom=check

H10 C4 O2

0 1

