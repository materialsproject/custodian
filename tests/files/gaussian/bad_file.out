 Entering Gaussian System, Link 0=g16
 Initial command:
 /gpfs/software/gaussian/g16/l1.exe "/gpfs/scratch/ratwi/tests/Custodian/bad_file/new/Gau-1220.inp" -scrdir="/gpfs/scratch/ratwi/tests/Custodian/bad_file/new/"
 Entering Link 1 = /gpfs/software/gaussian/g16/l1.exe PID=      1221.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                25-Mar-2024 
 ******************************************
 %chk=Checkpoint.chk
 %mem=48GB
 %NProcShared=28
 Will use up to   28 processors via shared memory.
 Bad file opened by FileIO:  Unit=2 NFiles=         0 MaxFFi=     10000.
 FileIO: IOper= 9 IFilNo(1)=     2 Len=           0 IPos=           0 Q=         83421640


 dumping /fiocom/, unit = 1 NFiles =     1 SizExt =    524288 WInBlk =   1048576
                   defal = F LstWrd =     2097152 FType=2 FMxFil=10000

 Number              0
 Base          1048576
 End           2097152
 End1          2097152
 Wr Pntr       1048576
 Rd Pntr       1048576
 Length        1048576
 Int.Num             1
 Bad file opened by FileIO:  Unit=2 NFiles=         0 MaxFFi=     10000.
 FileIO: IOper= 9 IFilNo(1)=     2 Len=           0 IPos=           0 Q=         83421640
 Error termination in NtrErr:
 NtrErr Called from FileIO.
